# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/xnnpack/hotfix-lib64.patch
# Copyright (C) 2026 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- XNNPACK-124d961219d5486cda611908be55f9b5ba82ac8c/CMakeLists.txt.vanilla	2026-08-07 19:32:37.944786087 +0300
+++ XNNPACK-124d961219d5486cda611908be55f9b5ba82ac8c/CMakeLists.txt	2026-08-07 19:34:01.678998965 +0300
@@ -1272,7 +1272,7 @@
       "${CMAKE_BINARY_DIR}/cpuinfo")
   ELSE()
     ADD_LIBRARY(cpuinfo SHARED IMPORTED)
-    FIND_LIBRARY(CPUINFO_LIBRARY cpuinfo PATHS "${CPUINFO_SOURCE_DIR}/lib")
+    FIND_LIBRARY(CPUINFO_LIBRARY cpuinfo PATHS "${CPUINFO_SOURCE_DIR}/lib64")
     IF(NOT CPUINFO_LIBRARY)
       MESSAGE(FATAL_ERROR "Cannot find cpuinfo")
     ENDIF()
@@ -1297,7 +1297,7 @@
       "${CMAKE_BINARY_DIR}/pthreadpool")
   ELSE()
     ADD_LIBRARY(pthreadpool SHARED IMPORTED)
-    FIND_LIBRARY(PTHREADPOOL_LIBRARY pthreadpool PATHS "${PTHREADPOOL_SOURCE_DIR}/lib")
+    FIND_LIBRARY(PTHREADPOOL_LIBRARY pthreadpool PATHS "${PTHREADPOOL_SOURCE_DIR}/lib64")
     IF(NOT PTHREADPOOL_LIBRARY)
       MESSAGE(FATAL_ERROR "Cannot find pthreadpool")
     ENDIF()

